Documentation/using-rkt-with-systemd.md

```diff
@@ -202,6 +202,86 @@ Jul 30 12:24:50 locke-work systemd[1]: Listening on My socket-activated app's so
 
 Now, a new connection to port 8080 will start your container to handle the request.
 
+### Bidirectionally proxy local sockets to another (possibly remote) socket.
+
+`rkt` also supports [socket-proxyd service][systemd-socket-proxyd]. So, like in
+ socket activation systemd will listen on a port on behalf of a container, and start the container when receiving a connection. It could be useful to provide socket activation for environment that natively doesn't support it. One of the
+ example is the `KVM` flavor in `stage1`.
+
+To make socket proxyd work, create network template with three units, like in example below (with redis app and ptp network template in `/etc/rkt/net.d/ptp0.conf`):
```

Comment:
"create a network template consisting of three units, like the example below. This example uses the redis app and the PTP network template in `/etc/rkt/net.d/ptp0.conf`: